examples/quine.html

```diff
@@ -3,10 +3,13 @@
 <head>
 <meta charset="utf-8" />
 <title>Making Quines Prettier</title>
-<!-- The defer is not necessary for autoloading, but is necessary for the
-     script at the bottom to work as a Quine. -->
-<script src="https://cdn.rawgit.com/google/code-prettify/master/loader/run_prettify.js?autoload=true&amp;skin=sunburst&amp;lang=css" defer="defer"></script>
-<style>.operative { font-weight: bold; border:1px solid yellow }</style>
+<!-- The defer attribute is not necessary for autoloading, but is necessary
+     for the script at the bottom to work as a Quine. -->
+<script src="https://cdn.rawgit.com/google/code-prettify/master/loader/run_prettify.js?autoload=true&amp;skin=sunburst&amp;lang=css" defer></script>
```

Comment:
@kevin-brown oh I just understood you're last point there! In case anyone else didn't know, RawGit CDN caching for master branch URLs now expires after 24 hours: https://github.com/rgrove/rawgit/issues/134#issuecomment-271773725